packages/nx/src/project-graph/project-graph-error.ts

```diff
@@ -0,0 +1,74 @@
+import { DaemonProjectGraphError } from '../daemon/daemon-project-graph-error';
```

Comment:
This can move into `error-types.ts`

Reply:
moved DaemonProjectGraphError into `error-types.ts`